### PR TITLE
add creek70

### PR DESCRIPTION
**Pull request by spbgzh**
## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/19789

## Checklist

- [x] The VIA support for this keyboard is in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/wuque/creek70/creek70.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  "name": "Creek 70",
+  "vendorId": "0x916D",
+  "productId": "0x0001",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
+    {
+      "label": "Lighting",
+      "content": [
+        {
+          "label": "Rigth_Color",
```
Comment (Xelus22) [inline v3/wuque/creek70/creek70.json:11]:
```suggestion
        {
          "label": "Right Color",
```

File: v3/wuque/creek70/creek70.json [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+              "type": "dropdown",
+              "content": ["id_qmk_rgblight_effect", 3, 2],
+              "options": [
+                "All Off",
+                "Solid Color",
+                "Breathing",
+                "Rainbow Moving Chevron",
+                "Rainbow Beacon",
+                "Rainbow Pinwheels"
+              ]
```
Comment (Xelus22) [inline v3/wuque/creek70/creek70.json:30]: From your keyboard `config.h` there are only Breathing, Rainbow Mood and Static gradient. Is this correct?

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/wuque/creek70/creek70.json (hunk 2 above)
Comment (spbgzh) [reply v3/wuque/creek70/creek70.json:30]: yes, different kinds of Rainbow affection is for rgb matrix.

Comment (Xelus22) [reply v3/wuque/creek70/creek70.json:30]: However you do not have RGB matrix at all? You only have RGB Light.

image

You only have 
```
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
```

Comment (spbgzh) [reply v3/wuque/creek70/creek70.json:30]: I added long time ago, seill no one check. ( https://github.com/qmk/qmk_firmware/pull/19858

**Comment by yiancar** [comment]
I will close this PR for now please ping us when QMK is merged and I will reopen it.

**Pull request closed without merging**